start.sh

```diff
@@ -152,6 +163,39 @@ show_help() {
     echo "  -n, --name                 Specify the project name"
 }
 
+
+# Function to get user selection and return the hex value of the selected color
+select_color() {
+    # Define color options
+    declare -a color_names=("White" "Light Gray" "Light Yellow" "Light Blue" "Light Green" )
```

Comment:
Using indexed arrays to support the older version of Bash that ships with macs. Associative arrays (similar to maps) are not available.